source/blog/2018-07-13-the-ember-times-issue-55.md

```diff
@@ -0,0 +1,136 @@
+---
+title: The Ember Times - Issue No. 55
+author: Kenneth Larsen, Amy Lam, Miguel Braga Gomes, Ryan Mark, Sivakumar Kailasam, Jessica Jordan, Alon Bukai
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
+alias : "blog/2018/07/13-the-ember-times-issue-55.html"
+responsive: true
+---
+
+nuqneH Emberistas! 🐹
```

Comment:
That's a fun greeting :sparkles:

Reply:
Tried to think of the geekiest greeting :)